Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:18.04
+RUN set -xe \
+    && apt-get update \
+    && apt-get install -y curl gcc g++ \
+    && apt-get install -y python3-dev \
+    && apt-get install -y libffi-dev \
+    && apt-get install -y libssl-dev \
+    && apt-get install -y python3-pip \
+    && apt-get install -y git
+RUN pip3 install --upgrade pip
+
+ADD . /selected-tests
+WORKDIR /selected-tests
+RUN pip3 install .
+RUN pip install gunicorn==19.9.0
```

Comment:
Is this pip intentionally different from the one above?

Reply:
Yes, gunicorn is not a requirement in setup.py (the requirements in setup.py are required for the package to run and will get installed with `pip3 install .`), but gunicorn is needed for the Docker container to run (since it gets called in `scripts/server.sh`), so I needed to install it in the Docker image.

Reply:
Oh, you meant pip vs. pip3 :). Switching to pip3.